composer.json

```diff
@@ -66,6 +66,17 @@
         ]
     },
     "extra": {
+        "installer-types": ["bower-asset", "npm-asset", "ramsaltmedia-module", "ramsaltmedia-theme"],
```

Comment:
It's doubling the installer-paths. Could you check this?

Reply:
So you're suggesting to mix this into `installer-paths` directly? like `"web/themes/ramsaltmedia/{$name}": ["type:ramsaltmedia-module"],` ?

Reply:
See the resulting composer.json. It has two installer paths sections then, which is wrong.
![Bildschirmfoto 2022-02-11 um 15 19 13](https://user-images.githubusercontent.com/1475847/153608046-a72726fc-553e-4c78-b473-b0af2b280038.png)

Reply:
Idk how that even happened but removed the obviously older `installer-paths` at the bottom